src/ubuntu/16.04/crossdeps/Dockerfile

```diff
@@ -2,6 +2,10 @@ FROM mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-16.04-coredeps
 
 # Install the base toolchain we need to build anything (clang, cmake, make and the like).
 RUN apt-get update \
+    && apt-get install -y apt-transport-https ca-certificates gnupg software-properties-common wget \
```

Comment:
Could you format this as done below where each package is on a separate line?  This helps the readability so that you can more easily pick out which packages are included in the image.  Additionally wget can be removed below since it is now included here.